packages/database/data-migrations/20240611153820_pricing_v2/data-migration.ts

```diff
@@ -0,0 +1,172 @@
+          },
+        });
+      }
+
+      console.log("Moved all organizations without billing to free plan");
+
+      const orgsWithBilling = await tx.organization.findMany({
+        where: {
+          billing: {
+            path: ["stripeCustomerId"],
+            not: Prisma.AnyNull,
+          },
+        },
+      });
+
+      console.log(`Found ${orgsWithBilling.length} organizations with billing information`);
+
+      for (const org of orgsWithBilling) {
+        const billing = org.billing as TOrganizationBillingLegacy;
+
+        // @ts-expect-error
+        if (billing.plan && typeof billing.plan === "string") {
+          // no migration needed, already following the latest schema
+          continue;
+        }
+
+        console.log({ billing });
+
+        if (
+          (billing.features.linkSurvey?.status === "active" && billing.features.linkSurvey?.unlimited) ||
+          (billing.features.inAppSurvey?.status === "active" && billing.features.inAppSurvey?.unlimited) ||
+          (billing.features.userTargeting?.status === "active" && billing.features.userTargeting?.unlimited)
+        ) {
+          await tx.organization.update({
+            where: {
+              id: org.id,
+            },
+            data: {
+              billing: {
+                plan: "enterprise",
+                limits: {
+                  monthly: {
+                    responses: null,
+                    miu: null,
+                  },
+                },
+              },
+            },
+          });
+
+          console.log("Updated org with unlimited to enterprise plan: ", org.id);
+          continue;
+        }
+
+        if (billing.features.linkSurvey.status === "active") {
+          await tx.organization.update({
+            where: {
+              id: org.id,
+            },
+            data: {
+              billing: {
+                plan: "startup",
+                limits: {
+                  monthly: {
+                    responses: 2000,
+                    miu: 2500,
+                  },
+                },
+              },
+            },
+          });
+
+          console.log("Updated org with an active link survey plan to the new startup plan: ", org.id);
+          continue;
+        }
+
+        if (
+          billing.features.inAppSurvey.status === "active" ||
+          billing.features.userTargeting.status === "active" ||
+          billing.features.multiLanguage.status === "active"
+        ) {
+          await tx.organization.update({
+            where: {
+              id: org.id,
+            },
+            data: {
+              billing: {
+                plan: "free",
+                limits: {
+                  monthly: {
+                    responses: 500,
+                    miu: 1000,
+                  },
+                },
+              },
+            },
+          });
+
+          console.log("Updated org to free plan: ", org.id);
+          continue;
+        }
```

Comment:
Can we print the organization ID if we forgot someone after these checks that wasn't migrated?